packages/belos/src/BelosCGIter.hpp

```diff
@@ -176,7 +176,13 @@ class CGIter : virtual public CGIteration<ScalarType,MV,OP> {
 
   //! Get the norms of the residuals native to the solver.
   //! \return A std::vector of length blockSize containing the native residuals.
-  Teuchos::RCP<const MV> getNativeResiduals( std::vector<MagnitudeType> * /* norms */ ) const { return R_; }
+  Teuchos::RCP<const MV> getNativeResiduals( std::vector<MagnitudeType> * norms ) const {
+    if (foldConvergenceDetectionIntoAllreduce_) {
```

Comment:
This is conditional on the type of norm being requested, see the same method in CGSingleRedIter.  Is there a reason it should be different here?

Reply:
Good catch!